source/_docs/blueprint/selectors.markdown

```diff
@@ -18,27 +18,32 @@ The following selectors are currently available:
 - [Action selector](#action-selector)
 - [Add-on selector](#add-on-selector)
 - [Area selector](#area-selector)
+  - [Example area selectors](#example-area-selectors)
```

Comment:
@raman325 @c0ffeeca7 

This not a traditional TOC, seems this unrelated change sneaked in. Could you take a look?

Reply:
Right. Let me remove it.

Reply:
Hmm I'm not sure how this happened. Perhaps I started from next and then switched to current? Not sure